
==================[13.04.2023 13:12:42]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:12:42]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:12:42]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:12:42]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:12:59]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:13:00]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:13:00]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:13:00]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:13:00]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:13:13]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:13:13]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:13:13]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:13:13]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:14]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:14]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:14]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:14]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:15]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:16]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:17]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[13.04.2023 13:14:17]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[13.04.2023 13:14:17]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[13.04.2023 13:14:17]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at int MySql.Data.MySqlClient.Driver.GetResult(int statementId, ref int affectedRows, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[13.04.2023 13:14:17]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '0' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=====
==================[13.04.2023 13:28:51]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[13.04.2023 13:28:51]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[13.04.2023 13:28:51]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[13.04.2023 13:28:51]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[13.04.2023 13:28:51]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[13.04.2023 13:28:51]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[13.04.2023 13:28:51]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[13.04.2023 13:28:51]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[13.04.2023 13:28:51]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[13.04.2023 13:28:51]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sock
==================[13.04.2023 13:29:22]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[13.04.2023 13:29:22]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[13.04.2023 13:29:22]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[13.04.2023 13:29:22]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[13.04.2023 13:29:22]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[13.04.2023 13:29:22]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[13.04.2023 13:29:22]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[13.04.2023 13:29:22]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[13.04.2023 13:29:22]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[13.04.2023 13:29:22]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sock
==================[14.04.2023 13:27:30]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[14.04.2023 13:27:30]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[14.04.2023 13:27:30]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[14.04.2023 13:27:30]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306)
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at void System.Threading.Tasks.Task.ThrowIfExceptional(bool includeTaskCanceledExceptions)
=========================================================

==================[14.04.2023 13:27:30]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[14.04.2023 13:27:30]==================
System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'. Не удается найти указанный файл.
File name: 'C:\Users\Kamal\Desktop\server-files\dotnet\resources\roleplay\Release\MySql.Data.resources.dll'
   at void System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, string ilPath, string niPath, ObjectHandleOnStack retAssembly)


=========================================================

==================[14.04.2023 13:27:30]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
=========================================================

==================[14.04.2023 13:27:30]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[14.04.2023 13:27:30]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at Task System.Net.Sockets.TcpClient.ConnectAsync(string host, int port)+(IAsyncResult asyncResult) => { } [1]
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   at void MySql.Data.MySqlClient.NativeDriver.Open()+(Exception ex) => { }
   at void AggregateException.Handle(Func<Exception, bool> predicate)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[14.04.2023 13:27:30]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts.
 ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение. 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw(Exception source)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sock